lib/y2snapper_common.pm

```diff
@@ -11,6 +11,19 @@ use testapi;
 use utils;
 use version_utils;
 
+=head2 y2snapper_create_initial_snapshot
+
+ y2snapper_create_initial_snapshot();
+
+Create initial snapshot using snapper to ensure to have a clear diff
+before creating the snapshot in the UI.
+
+=cut
+sub y2snapper_create_initial_snapshot {
+    assert_script_run('snapper create --description "Amazing Snapshot" --userdata "a=0"');
```

Comment:
Not a big deal. But the description could be more descriptive here, right?

Reply:
The important thing is the function documentation, `--description` param does not matter, it could be anything.

Reply:
> is this still WIP? The VR for SLE12 all fail

I missed the needle :( Creating it.

Reply:
i know. just to reflect the purpose of the snapshot. not an issue anyway